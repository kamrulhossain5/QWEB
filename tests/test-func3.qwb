function none printem(int a, int b, int c, int d){
  print(a);
  print(b);
  print(c);
  print(d);
}

function int main(){
  printem(15,17,192,8);
}
