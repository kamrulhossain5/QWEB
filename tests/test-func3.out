15
17
192
8
